File: Programar em C#/07-Mais-sobre-as-Strings.cs
// Programar em Python #07 - Mais sobre as Strings

using System;

namespace Base {
    class MaisSobreAsStrings {
        public void Run() {
            string primeiroNome = "Nelson";
            string ultimoNome = "Silva";
            int idade = 21;

            // Console.WriteLine("Nome: " + primeiroNome + " " + ultimoNome + "\nIdade: " + idade);
            // Console.WriteLine("Nome: {0} {1}\nIdade: {2}", primeiroNome, ultimoNome, idade);
            Console.WriteLine($"Nome: {primeiroNome} {ultimoNome}\nIdade: {idade}");
        }
    }
}

File: Programar em Java/07-Mais-sobre-as-Strings.java
// Programar em Java #07 - Mais sobre as Strings

package com.caffeinealgorithm.programaremjava;

public class MaisSobreAsStrings {
    public void Run() {
        String primeiroNome = "Nelson";
        String ultimoNome = "Silva";
        int idade = 21;

        System.out.println("Nome: " + primeiroNome + ' ' + ultimoNome + "\nIdade: " + idade);
        System.out.println(new StringBuilder().append("Nome: ").append(primeiroNome).append(' ').append(ultimoNome).append("\nIdade: ").append(idade));
    }
}